src/custom-apps/components/WebhookSubscriptionQuery/globals.css

```diff
@@ -0,0 +1,11 @@
+.graphiql-editor {
+  font-size: 20px;
+}
+
+#graphiql-session > div:last-child {
+  display: none !important;
+}
+
+.graphiql-execute-button, .graphiql-sidebar:last-child {
+  display: none;
+}
```

Comment:
**issue**: this is a very forcing way to solve a local problem, i'm pretty sure there is another way to do it, but if there is none please provide a comment explaining why this solution was used

Reply:
@Droniu GraphiQL packages has its own way of applying styles. We were not sure how to combine that with Macaw UI. What would be the idiomatic way in this codebase?